Makefile

```diff
@@ -3,7 +3,7 @@ SERVER=""
 
 install:
 	python3.6 -m ensurepip
-	pkg install -q -y libgit2
+	pkg install -q -y libgit2 cython
```

Comment:
we *could* install cython3 however, from pkg, it would make things a lot less painful.